net-mail/vpopmail/vpopmail-5.4.33-r4.ebuild

```diff
@@ -0,0 +1,300 @@
+	"${FILESDIR}"/${PN}-lazy.patch
+	"${FILESDIR}"/${PN}-vpgsql.patch
+	"${FILESDIR}"/${PN}-double-free.patch
+	"${FILESDIR}"/${PN}-5.4.33-vdelivermail-add-static.patch
+	"${FILESDIR}"/${PN}-5.4.33-fix-those-vfork-instances-that-do-more-than-exec.patch
+	"${FILESDIR}"/${PN}-5.4.33-remove-unneeded-forward-declaration.patch
+	"${FILESDIR}"/${PN}-5.4.33-clean-up-calling-maildrop.patch
+	"${FILESDIR}"/${PN}-5.4.33-fix-S-tag-in-case-spamassassin-changed-the-file-size.patch
+	"${FILESDIR}"/${PN}-5.4.33-strncat.patch
+)
+DOCS=(
+	ChangeLog
+	doc/AUTHORS
+	doc/FAQ
+	doc/INSTALL
+	doc/README.activedirectory
+	doc/README.authvchkpw
+	doc/README.filelocking
+	doc/README.ipaliasdomains
+	doc/README.ldap
+	doc/README.maildrop
+	doc/README.mysql
+	doc/README.onchange
+	doc/README.oracle
+	doc/README.pgsql
+	doc/README.qmail-default
+	doc/README.quotas
+	doc/README.roamingusers
+	doc/README.spamassassin
+	doc/README.sybase
+	doc/README.vdelivermail
+	doc/README.vlimits
+	doc/README.vpopmaild
+	doc/README.vqmaillocal
+	doc/UPGRADE
+)
+HTML_DOCS=(
+	doc/doc_html/vpopmail5.abw
+	doc/doc_html/vpopmailapi.png
+	doc/doc_html/vpopmail.html
+	doc/man_html/clearopensmtp.html
+	doc/man_html/index.html
+	doc/man_html/vaddaliasdomain.html
+	doc/man_html/vadddomain.html
+	doc/man_html/vadduser.html
+	doc/man_html/vchkpw.html
+	doc/man_html/vconvert.html
+	doc/man_html/vdeldomain.html
+	doc/man_html/vdelivermail.html
+	doc/man_html/vdeluser.html
+	doc/man_html/vpasswd.html
+	doc/man_html/vpopbull.html
+	doc/man_html/vsetuserquota.html
+)
+
+# This makes sure the variable is set, and that it isn't null.
+VPOP_DEFAULT_HOME="/var/vpopmail"
+
+vpopmail_set_homedir() {
+	VPOP_HOME=$(egethome vpopmail)
+	if [[ -z "${VPOP_HOME}" ]]; then
+		eerror "vpopmail's home directory is null in passwd data!"
+		eerror "You probably want to check that out."
+		eerror "Continuing with default."
+		VPOP_HOME="${VPOP_DEFAULT_HOME}"
+	else
+		einfo "Setting VPOP_HOME to: $VPOP_HOME"
+	fi
+}
+
+pkg_setup() {
+	enewgroup vpopmail 89
+	enewuser vpopmail 89 -1 ${VPOP_DEFAULT_HOME} vpopmail
+	upgradewarning
+}
+
+src_prepare() {
+	default
+
+	echo 'install-recursive: install-exec-am' \
+		>>"${S}"/Makefile.am
+
+	# fix maildir paths
+	sed -i -e 's|Maildir|.maildir|g' \
+		vchkpw.c vconvert.c vdelivermail.c \
+		vpopbull.c vpopmail.c vqmaillocal.c \
+		vuserinfo.c maildirquota.c || die
+
+	# remove vpopmail advertisement
+	sed -i -e '/printf.*vpopmail/s:vpopmail (:(:' \
+		vdelivermail.c vpopbull.c vqmaillocal.c || die
+
+	# automake/autoconf
+	mv -f "${S}"/configure.{in,ac} || die
+	sed -i -e 's,AM_CONFIG_HEADER,AC_CONFIG_HEADERS,g' \
+	        configure.ac || die
+
+	# _FORTIFY_SOURCE
+	sed -i \
+		-e 's/\(snprintf(\s*\(LI->[a-zA-Z_]\+\),\s*\)[a-zA-Z_]\+,/\1 sizeof(\2),/' \
+		vlistlib.c || die
+
+	eautoreconf
+	ht_fix_file cdb/Makefile
+}
+
+src_configure() {
+	vpopmail_set_homedir
+
+	local authopts
+	if use mysql; then
+		incdir=$(mysql_config --variable=pkgincludedir)
```

Comment:
Needs to die if it cant be executed it'll ruin the ebuild config. Similarily to one below and pg_config.